index.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Mobile First Project</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.3/css/all.min.css"/>
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&family=Roboto&display=swap" rel="stylesheet">
+</head>
+<body>
+
+    <header class="main-header">
+
+            <nav class="header">
+
+                <div class="name"><h1>Paul Orife</h1></div>
+                <div class="menu"><i class="fas fa-bars"></i></div>
+                
+            </nav>
+    </header>
+
+    <main class="main1">
+          <section class="section1">
+            <div class="intro">
+                <h1>Hey There. <br>
+                    I'm Paul
+                </h1>
+            </div>
+
+            <div class="profession"><h2>I'am a Software Developer</h2></div>
+
+            <article class="services">
+                <p>
+                    I can help you build a product , feature or <br>
+                     website Look through some of my work <br>
+                      and experience! If you like what you see <br>
+                     and have a project you need coded, <br>
+                    don’t hestiate to contact me. <br>
+                </p>
+            </article>
+
+            <div>
+                <ul class="logo">
+                    <li><a href="#"><img src="./image/github.png" alt=""></a></li>
+                    <li><a href="#"><img src="./image/linkedln.png" alt=""></a></li>
+                    <li><a href="#"><img src="./image/angellist.png" alt=""></a></li>
+                    <li><a href="#"><img src="./image/twitter.png" alt=""></a></li>
+                    <li><a href="#"><img src="./image/madiu.png" alt=""></a></li>
```

Comment:
- [ ] **[OPTIONAL]:** You did a great job adding `<a>` tags as the previous reviewer requested! 🥇 In order to make them useful, I suggest you include your social media links inside the `href` attribute of each link. It's okay if you haven't worked on your social media yet, but this is a reminder to update these links in the future! 🤓 🛠️